src/Couchbase.Lite.Shared/API/Sync/Replicator.cs

```diff
@@ -296,6 +303,10 @@ private static void OnDocEnded(C4Replicator* repl, bool pushing, IntPtr numDocs,
                 replicator.OnDocEnded(documentReplications, pushing);
             });
 
+            replicator?.DispatchQueue.DispatchAsync(() =>
+            {
+                replicator.OnDocEndedWithConflict(replicatedDocumentsContainConflict, pushing);
+            });
```

Comment:
It might be better to have both of these calls inside of one dispatch async instead of two, but it's up to you.